data/duMasdesBourbouxetal2020/BAO_for_external_use/README

```diff
@@ -0,0 +1,34 @@
+1. The official eBOSS DR16 scans can be found here: https://svn.sdss.org/public/data/eboss/DR16cosmo/tags/v1_0_0/likelihoods/
```

Comment:
@andreicuceu - I would clarify that in that link they can find the likelihoods used in the "DR16 cosmology paper" (and add the arXiv number to Alam et al. 2020).

I would also clarify in the README which are the scans used in the DR16 Lya BAO analysis, since these are not the same.